packages/dds/tree/src/feature-libraries/index.ts

```diff
@@ -128,5 +130,17 @@ export { TreeChunk, chunkTree, buildChunkedForest, defaultChunkPolicy } from "./
 
 // Split into separate import and export for compatibility with API-Extractor.
 import * as SchemaAware from "./schema-aware";
-import * as FieldKinds from "./defaultFieldKinds";
-export { SchemaAware, FieldKinds };
+import * as FieldKindsOriginal from "./defaultFieldKinds";
+export { SchemaAware };
+
+// Export subset of FieldKinds in an API-Extractor compatible way:
+import { FieldEditor, FieldKind, Multiplicity } from "./modular-schema";
```

Comment:
I think this import might be causing the import in the api file. The comment says export so is it supposed to be an export?

Reply:
Oh never mind, I see it's being used below. Might want to check if the import here is the cause and move FieldKinds to a separate file if it is.

Reply:
I tried a separate file, and it had no effect. I did find a way to simplify this a bit (and avoid allocating an extra object here) but it also had no effect in the api extractor results.